rlinf/config.py

```diff
@@ -654,7 +654,6 @@ def validate_embodied_cfg(cfg):
     cfg.algorithm.num_group_envs = (
         cfg.algorithm.num_group_envs // stage_num // env_world_size
     )
-    cfg.env.eval.num_envs = cfg.env.eval.num_envs // stage_num // env_world_size
```

Comment:
Why can this be removed without changing all the other envs? This causes `num_envs` to have different meanings in train an eval. Fixing this requires major refactor to everything. It's not suitable to do this in this PR.